src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -49,11 +49,40 @@ export class ArtworkSidebarCommercial extends React.Component<
         ) : (
           this.renderEditions()
         )}
+        {!artwork.is_sold &&
```

Comment:
I see you are trying to account for the revert that we are planning to do in metaphysics. That is cool. But I think the check here should also include the check if the work is still set up for ecommerce before displaying those fields. It's possible now that partner set up the work for ecommerce - setting up shipping fields and then wend back and excluded the work from ecommerce. The shipping info is still on the work, but I believe in this case we should shide this information as well. Those kinda confusing rules is part of the reason why I wanted to keep it all in Metaphysics. And here just display it if it's present:)